bin/picca_Pk1D_average_mean.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+"""Compute the averaged 1D power spectrum
+"""
+
+import sys, glob, argparse, ast
+from picca.pk1d import postproc_pk1d
+from picca.utils import userprint
+
+
+def main(cmdargs):
+    """Compute the averaged 1D power spectrum averages"""
+
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Compute the averaged 1D power spectrum averages",
+    )
+
+    parser.add_argument(
+        "--in-dir",
+        type=str,
+        default=None,
+        required=False,
+        help="String to glob all mean p1d to average",
```

Comment:
`in-dir` would normally mean you put in a directory name and a predefined glob will be running within that dir, here it looks like a full glob-expression needs to be supplied, which I think is ok as well, but maybe should use a different name...